nrf52840-hal/src/ieee802154.rs

```diff
@@ -38,11 +35,23 @@ pub const DEFAULT_TXPOWER: TxPower = TxPower::_0dBm;
 /// Default Start of Frame Delimiter = `0xA7` (IEEE compliant)
 pub const DEFAULT_SFD: u8 = 0xA7;
 
+/// note: picked trough trial & error
+pub const DEFAULT_ED_THRESHOLD: u8 = 0x50;
+
+/// Default Clear Channel Assessment method = Carrier sense
+pub const DEFAULT_CCA: Cca = Cca::CarrierSense;
+
 // TODO expose the other variants in `pac::CCAMODE_A`
 /// Clear Channel Assessment method
 pub enum Cca {
     /// Carrier sense
     CarrierSense,
+    /// Energy above the Energy Detection threshold. Energy measurements above this value mean
+    /// that the channel is assumed to be busy.
+    /// Note the the measurement range is 0..0xFF - where 0 means that the received power was
```

Comment:
I would move this second half of the documentation to the documentation of the `ed_threshold` field.